### ISSUE TITLE
Question on multiplying constant to sphereR_H, sphereR_N and sphereR_N 

### ISSUE THREAD

**Issue by lizhenstat** [ydwen/opensphere#18]
Hi, thanks for sharing the code of your great work on SphereFace Revived. I have a related question: 
when using the three proposed normalization methods, why do you scaling a constant on the cross entropy loss
in [sphereR_N](https://github.com/ydwen/opensphere/blob/main/model/head/spherefacer.py#L68),[sphereR_H](https://github.com/ydwen/opensphere/blob/main/model/head/spherefacer.py#L100) and [sphereR_S](https://github.com/ydwen/opensphere/blob/main/model/head/spherefacer.py).

Any help would be appreciated, thanks!

**Comment by ydwen** [comment]
lw is loss weight, controlling the loss scale.

**Comment by lizhenstat** [comment]
thanks a lot,  I found an [interesting answer](https://stats.stackexchange.com/questions/346299/whats-the-effect-of-scaling-a-loss-function-in-deep-learning) why scaling loss does affect the training result. 
It seems that scaling the loss under SGD and no regularization equals scaling the learning rate.
